web/client/plugins/__tests__/TOC-test.jsx

```diff
@@ -12,7 +12,7 @@ import ReactDOM from 'react-dom';
 import TOCPlugin from '../TOC';
 import { getPluginForTest } from './pluginsTestUtils';
 
-describe('TOCPlugin Plugin', () => {
+describe.only('TOCPlugin Plugin', () => {
```

Comment:
we should ban the use of describe.only :-)

Reply:
done it